zulip_bots/zulip_bots/bots/file_uploader/file_uploader.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from pathlib import Path
+
+class FileUploaderHanlder(object):
+    def usage(self) -> str:
+        return (
+            'This interactive bot used to upload files, images, etc to Zulip server.\n'
+            'Specify the file path you want to upload and file will be upload to Zulip server'
+        )
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        path = Path(message['content'].strip())
+        if not path.is_file():
+            bot_handler.send_reply(message, 'File not found')
```

Comment:
Consider adding the filename here, as in the failed-upload case, so the user can be sure of what path failed? Also perhaps in the case of multiple upload commands overlapping.

Reply:
@neiljp changed message to `File {} not found`